ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/discovery/internal/PeerDiscoveryControllerTest.java

```diff
@@ -718,8 +716,8 @@ public void shouldNotBondWithBlacklistedPeer() {
         MockPacketDataFactory.mockNeighborsPacket(discoPeer, otherPeer, otherPeer2);
     controller.onMessage(neighborsPacket, discoPeer);
 
-    verify(controller, times(0)).bond(otherPeer);
-    verify(controller, times(1)).bond(otherPeer2);
+    verify(controller, times(0)).bond(eq(otherPeer));
+    verify(controller, times(1)).bond(eq(otherPeer2));
```

Comment:
nit: I don't think we need the `eq` here.  The default is equals comparison so you typically only use `eq` if you need to use other matchers for some params (because all params must use matchers if any do).

Reply:
Removed